project_template/bin/ci

```diff
@@ -3,5 +3,5 @@ set -e
 
 yarn lint
 yarn test:unit
-yarn test:e2e
+yarn test:e2e -- --env ci
```

Comment:
This will trigger a warning in Yarn 1.0 and up: `From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.`

It would be best to drop the extra `-- `.